src/Worker.php

```diff
@@ -85,11 +96,14 @@ private function tick($queue)
             $result = call_user_func($handler, $message);
 
             $this->event->finish($message);
+            $this->logger->info(sprintf('`%s` job finished', $handle));
 
             if ($result === false) {
+                $this->logger->notice('shutting down');
```

Comment:
Do we maybe want to include the name of the handler responsible for terminating the process here?